VirtoCommerce.Storefront/Domain/Security/PollingApiUserChangeToken.cs

```diff
@@ -1,31 +1,36 @@
 using System;
+using System.Linq;
 using System.Threading;
 using Microsoft.Extensions.Primitives;
 using VirtoCommerce.Storefront.AutoRestClients.PlatformModuleApi;
+using VirtoCommerce.Storefront.AutoRestClients.PlatformModuleApi.Models;
 using VirtoCommerce.Storefront.Model.Common;
-using VirtoCommerce.Storefront.Model.Security;
 
 namespace VirtoCommerce.Storefront.Domain.Security
 {
     public sealed class PollingApiUserChangeToken : IChangeToken
     {
         private readonly ISecurity _platformSecurityApi;
-        private DateTime _lastCheckedTimeUtc;
+        private static DateTime _previousChangeTimeUtcStatic;
+        private static DateTime _lastCheckedTimeUtcStatic;
         private readonly TimeSpan _pollingInterval;
-        private readonly User _user;
+
         private readonly object _lock = new object();
 
-        private PollingApiUserChangeToken(User user, ISecurity platformSecurityApi, TimeSpan pollingInterval)
+        static PollingApiUserChangeToken()
+        {
+            _previousChangeTimeUtcStatic = _lastCheckedTimeUtcStatic = DateTime.UtcNow;
+        }
+
+        private PollingApiUserChangeToken(ISecurity platformSecurityApi, TimeSpan pollingInterval)
         {
-            _user = user;
-            _lastCheckedTimeUtc = DateTime.UtcNow;
             _pollingInterval = pollingInterval;
             _platformSecurityApi = platformSecurityApi;
         }
 
-        public static IChangeToken CreateChangeToken(User user, ISecurity platformSecurityApi, TimeSpan pollingInterval)
+        public static IChangeToken CreateChangeToken(ISecurity platformSecurityApi, TimeSpan pollingInterval)
```

Comment:
Probably we could just use public constructor instead of static `CreateChangeToken` method.